client/stylesheets/abstracts/_colors.scss

```diff
@@ -3,3 +3,66 @@
 @import 'node_modules/@automattic/color-studio/dist/color-variables.scss';
 @import 'node_modules/@wordpress/base-styles/colors.native.scss';
 @import 'node_modules/@wordpress/base-styles/colors.scss';
+
+// WP-admin colors: https://make.wordpress.org/core/2021/02/23/standardization-of-wp-admin-colors-in-wordpress-5-7/
```

Comment:
Adding all standardised WP-admin colors to allow code throughout WooPayments to utilise.